.github/workflows/flake-maintenance.yml

```diff
@@ -0,0 +1,46 @@
+name: Flake maintenance
+
+on:
+  schedule:
+    - cron: "0 14 * * 1" # weekly at 14:00 UTC
+  workflow_dispatch:
+
+jobs:
+  check-flake:
```

Comment:
what is the purpose of this `check-flake` cron action? I am a little confused as it runs the `flake-checker-action` but then in the `update-flake-lock` job you also run `nix flake check` on the updated lock